students/lee_deitesfeld/lesson03/mailroom.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+
+import sys
+from operator import itemgetter
+
+lst_donors = [["Bob Jones", [63.00, 30.00]],
+            ["Sue Smith", [50.00]],
+            ["Joe Grimes", [102.00, 45.00, 70.00]],
+            ["Andrea Funk", [21.00, 21.00]],
+            ["Mickey Mouse", [500.00, 250.00, 100.00]]
+            ]
+
+prompt = "\n".join(("Please choose from below mailroom options:",
+          "1 - Send a Thank You",
+          "2 - Create a Report",
+          "3 - Quit",
+          ">>> "))
+
+def listof_donors():
+    '''Displays list of donor names in donor list'''
+    for name in lst_donors:
+        print(name[0])
+
+def report():
```

Comment:
This works well the first time through, but when adding a donor or donation, it breaks the second time through.  Try and debug and reach out if you'd like help.